docs/cli/build.md

```diff
@@ -84,3 +84,60 @@ ARG ADDITIONAL_PACKAGE
 RUN apk --no-cache add curl ${ADDITIONAL_PACKAGE} \  
 
 ```
+
+## 2.1 Build options examples
+
+Let's see some practical examples with the `build-option` flag.
+
+* Use [Pillow](https://pillow.readthedocs.io/en/5.2.x/) for image processing in your Python function
+
+Create function with
+
+```
+$ faas-cli new faas_black_and_white --lang python3 --prefix <your-docker-namespace>
+```
+
+Add `pillow` to `requirements.txt` .
+
+Copy some resource images to `faas_black_and_white/`.
+
+Edit `handler.py`:
+
+```python
+import os
+from PIL import Image
+
+def handle(req):
+    img = Image.open("function/" + req)
+
+    blackAndWhite = img.convert('1')
+
+    blackAndWhite.save("function/blackAndWhite.jpeg")
```

Comment:
I'm not keen on building images into the function file-system.

Please could this change to read the file from the request? Is that possible? If not, fetch it from a URL passed as as string.

Reply:
@ivanayov